src/RecurringSwap.sol

```diff
@@ -93,20 +102,29 @@ contract RecurringSwap is QuarkScript {
         }
 
         bytes32 hashedConfig = _hashConfig(config);
```

Comment:
Why do we use hashConfig here and not just isolated storage since it's based on script nonce?

Reply:
It's because a nonce can have multiple recurring swap orders. The hashConfig provides a level of isolation, on top of the existing nonce-isolation, if a user signs multiple orders for the same nonce. See this [test](https://github.com/compound-finance/quark-scripts/blob/ff0050165f462ec50bbd7d38335869adac30306d/test/RecurringSwap.t.sol#L443) for example.